### ISSUE TITLE
Evidence Model - I wish it indicated Lookup vs Inferred

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#713]
![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/404dacb0-c98c-4794-a33d-6c30069bd5a7)

I realize its a one-hop so I should infer this is lookup evidence, but I lose track and it would be helpful if this modal also indicated path type.

**Comment by dnsmith124** [comment]
@sstemann the arrow icon behind the edge is intended to communicate this, just confirming that this indication is too subtle in your opinion?

**Comment by gglusman** [comment]
It's clear when seeing both types of arrows (dashed vs. continuous) at the same time, but the distinction gets lost when expanding details.

**Comment by sstemann** [comment]
@dnsmith124 i dont think there is any change here.  while i can't replicate the request in MVP2 if you look at MVP1 which has one-hop support paths I think you could see where this request originated from

this is the EPC of the Lookup path
![image](https://github.com/user-attachments/assets/057927c1-94d4-4b3c-82b0-6f184aecaf9e)

this is the EPC of the one-hop inferred path
![image](https://github.com/user-attachments/assets/c108d866-0e72-46f9-a76d-6ee912d4cecf)

another example where the predicate changes slightly
this is the EPC of the lookup path
![image](https://github.com/user-attachments/assets/3a7de03b-98a1-4539-97c1-d66af1a82e2a)

this is the EPC of the one-hop inferred path
![image](https://github.com/user-attachments/assets/e64a7144-589d-48f1-aed1-2e9584e65099)

but as a user - when i'm looking at one-hop EPC, i dont know if its part of a support or not.

**Comment by sstemann** [comment]
i'm still wishing for more distinction.  After thinking about it further - if the modal said Evidence for Inferred Edge and then Evidence for Supporting Paths 

Here is an inferred edge:
![image](https://github.com/user-attachments/assets/407c8da3-b8a5-414c-8a41-e0d60128cf12)

Here is one of it's supporting path edges:
![image](https://github.com/user-attachments/assets/73bdcba1-e65a-4797-bdc4-9ca5ed0dbec7)

If the title had more info, I think it might help

**Comment by dnsmith124** [comment]
@sstemann this is actually a feature of the facelift that I ran out of time to implement in the previous sprint, I plan to implement it during the current sprint. Here's a screenshot of the design to show what it'll look like: 
![Screenshot 2024-08-30 at 10 03 33 AM](https://github.com/user-attachments/assets/1415e016-8499-4ca3-a014-01a5b6fd1676)

**Comment by sstemann** [comment]
@dnsmith124 this doesn't appear be in Hammerhead  https://ui.test.transltr.io/results?l=Age-related%20Macular%20Degeneration&i=MONDO:0005150&t=0&r=0&q=1c0b9542-9b10-4d64-bbd5-d7d712ab5f5a

this is support path 1D, 

![image](https://github.com/user-attachments/assets/df5999d3-bb71-4b2c-959e-33b34858b9d5)

**Comment by dnsmith124** [comment]
@sstemann That's correct, unfortunately I ran out of time to implement this during the current sprint. Adding it as a high priority item for the next one.
